src/bin/vip-import-media-status.js

```diff
@@ -34,7 +34,12 @@ command( {
 		'Export any file errors encountered to a JSON file instead of a plain text file',
 		false
 	)
-	.argv( process.argv, async ( arg, { app, env, exportFileErrorsToJson } ) => {
+	.option(
+		'saveErrorLog',
+		'If errors were encountered, save the error log file without prompting',
```

Comment:
Let's make this description a bit more concise?

```suggestion
		'Download file-error logs without prompting',
```